receiver/awscontainerinsightreceiver/internal/stores/kubeletutil/kubeletclient.go

```diff
@@ -18,12 +21,29 @@ import (
 
 var kubeletNewClientProvider = kubelet.NewClientProvider
 
+const kubeletCAPath = "/etc/kubernetes/kubelet-ca.crt"
+
 type KubeletClient struct {
 	KubeIP     string
 	Port       string
 	restClient kubelet.Client
 }
 
+func isFileExist(filePath string) bool {
+	_, err := os.Stat(filePath)
+	if err == nil {
+		// path exists
+		return true
+
+	} else if errors.Is(err, os.ErrNotExist) {
```

Comment:
This is technically same as err !=nil{return False} but I think this code makes it easier to the cases.